<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>60</y>
      <width>751</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="line_taskinput"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Priority:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dropdown_priority">
       <item>
        <property name="text">
         <string>Low</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/low.png</normaloff>Icons/low.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/medium.png</normaloff>Icons/medium.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>High</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>Icons/high.png</normaloff>Icons/high.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Due date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dropdown_doto">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Choose</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_addtask">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icons/add-task.png</normaloff>Icons/add-task.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_deletetask">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Icons/delete-task.png</normaloff>Icons/delete-task.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_todo">
       <property name="text">
        <string>Todo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_doing">
       <property name="text">
        <string>Doing</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_done">
       <property name="text">
        <string>Done</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="list_todo">
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="list_doing">
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QListWidget" name="list_done">
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="p1">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="d1">
         <property name="text">
          <string>Due date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="t1">
         <property name="text">
          <string>Task name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="p2">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="d2">
         <property name="text">
          <string>Due date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="t2">
         <property name="text">
          <string>Task name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="p3">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="d3">
         <property name="text">
          <string>Due date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="t3">
         <property name="text">
          <string>Task name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
